h/static/scripts/annotator/host.coffee

```diff
@@ -7,9 +7,9 @@ Guest = require('./guest')
 module.exports = class Host extends Guest
   constructor: (element, options) ->
     # Make a copy of all options except `options.app`, the app base URL.
-    appOpts = queryString.stringify(
-      Object.assign({}, options, {app: undefined})
-    )
+    appOpts = queryString.stringify({
```

Comment:
Given that we're now using `queryString.stringify` to (in effect) do `'config=' + encodeURIComponent(data)`, perhaps we don't need that dependency in the injector bundle any more?